File: astera/6/euler6.rb
def funct1
    sum1 = 0
    for i in (1..100)
        sum1 += i**2
    end
    return sum1
end

def funct2
    sum2 = (1..100).inject {|sum, i| sum + i}
    return sum2**2
end

diff = (funct1 - funct2).abs
puts diff

File: astera/6/euler6.py
def funct1():
	sum1 = 0
	for i in xrange(1,101):
		sum1 += i**2
	return sum1

def funct2():
	sum2 = sum (xrange(1,101))
	return sum2**2

diff = abs(funct1() - funct2())
print diff